sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -0,0 +1,1248 @@
+
+    power_t : double, optional, default 0.5
+        The exponent for inverse scaling learning rate.
+        It is used for updating learning_rate_ when it
+        is set to 'invscaling'. Only used when algorithm='sgd'.
+
+    verbose : bool, optional, default False
+        Whether to print progress messages to stdout.
+
+    warm_start : bool, optional, default False
+        When set to True, reuse the solution of the previous
+        call to fit as initialization, otherwise, just erase the
+        previous solution.
+
+    momentum : float, default 0.9
+        Momentum for gradient descent update.  Should be between 0 and 1. Only
+        used when algorithm='sgd'.
+
+    nesterovs_momentum : boolean, default True
+        Whether to use Nesterov's momentum. Only used when algorithm='sgd' and
+        momentum > 0.
+
+    early_stopping : bool, default False
+        Whether to use early stopping to terminate when loss is not improving.
+        If set to true, it will automatically set aside 10% of training data
+        as validation and termninate training when validation score is not
+        improving by more than tol for two consecutive epochs.
+        Only effective when algorithm='sgd' or 'adam'
+
+    validation_fraction : float, optional, default 0.1
+        The proportion of training data to set aside as validation set for
+        early stopping. Must be larger than or equal to 0 and less than 1.
+        Only used if early_stopping is True
+
+    beta_1 : float, optional, default 0.9
+        Exponential decay rate for estimates of first moment vector in adam,
+        should be in [0, 1). Only used when algorithm='adam'
+
+    beta_2 : float, optional, default 0.999
+        Exponential decay rate for estimates of second moment vector in adam,
+        should be in [0, 1). Only used when algorithm='adam'
+
+    epsilon : float, optional, default 1e-8
+        Value for numerical stability. Only used when algorithm='adam'
+
+    Attributes
+    ----------
+    `loss_` : float
+        The current loss value computed by the loss function.
+
+    `coefs_` : list, length n_layers - 1
+        The ith element in the list represents the weight matrix corresponding
+        to layer i.
+
+    `intercepts_` : list, length n_layers - 1
+        The ith element in the list represents the bias vector corresponding to
+        layer i + 1.
+
+    n_iter_ : int,
+        The current number of iterations the algorithm has ran.
+
+    n_layers_ : int
+        Number of layers.
+
+    `n_outputs_` : int
+        Number of outputs.
+
+    `out_activation_` : string
+        Name of the output activation function.
+
+    Notes
+    -----
+    MLPRegressor trains iteratively since at each time step
+    the partial derivatives of the loss function with respect to the model
+    parameters are computed to update the parameters.
+
+    It can also use regularizer as a penalty term added to the loss function
```

Comment:
It can also use regularizer -> something else. This sentence doesnt seem correct as is.